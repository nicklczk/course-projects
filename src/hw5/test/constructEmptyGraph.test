# Test to construct empty graph
CreateGraph graph0