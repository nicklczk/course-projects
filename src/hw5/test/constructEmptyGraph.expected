# Test to construct empty graph
created graph graph0